bgpd/rfapi/vnc_debug.c

```diff
@@ -95,6 +95,7 @@ DEFUN (no_debug_bgp_vnc,
        "<no debug|undebug> bgp vnc <rfapi-query|import-bi-attach|import-del-remote|verbose>",
```

Comment:
There is some history that allows 'undebug to be used'.
I didn't see sufficient reason to pull them out at this point in time.  I am fine with 'hiding' the command at some time in the future and pulling it out too, just wasn't my goal here.